src/components/Bars/Bars.jsx

```diff
@@ -276,7 +282,11 @@ const Bars = createClass({
 												pointKind={1}
 												color={_.get(colorMap, field, palette[(fieldIndex + colorOffset ) % palette.length])}
 											>
-												{`${_.get(legend, field, field)}: ${yFormatter(data[seriesIndex][field])}`}
+												{hasToolTipTitles ?
```

Comment:
is there a way to incorporate this into `yFormatter` or have a `tooltipFormatter` function that you pass in to do all of this?